src/content/guides/ecommerce-website.mdx

```diff
@@ -79,7 +80,7 @@ autonumber
 ```html
   <html>
   <head>
-    <script src="https://service.centrapay.com/ecommerce/centrapay.js"></script>
+    <script src="https://service.centrapay.com/ecommerce/centrapay.js?merchantId=test"></script>
```

Comment:
Is this right? Above you have `https://service.centrapay.com/ecommerce/centrapay.js?merchantId={merchantId}` but here you have `https://service.centrapay.com/ecommerce/centrapay.js?merchantId=test`

Reply:
@alexBinns I had done different variations but have now made them the same for clarity ✨